eva_submission/eload_validation.py

```diff
@@ -60,7 +61,19 @@ def mark_valid_files_and_metadata(self, merge_per_analysis):
             self.eload_cfg.query('validation', validation_task, 'forced', ret_default=False)
             for validation_task in self.all_validation_tasks
         ]):
-            self.eload_cfg.set('validation', 'valid', 'analyses', value=self.eload_cfg.query('submission', 'analyses'))
+            self.eload_cfg.set('validation', 'valid', 'analyses',
+                               value=copy.copy(self.eload_cfg.query('submission', 'analyses')))
+            # If the normalisation was successful and required then replace the valid submission file
+            # with the normalised one
+            for analysis_alias in self.eload_cfg.query('submission', 'analyses'):
+                normalised_vcf_file = []
+                for vcf_file in self.eload_cfg.query('submission', 'analyses', analysis_alias, 'vcf_files'):
+                    if self.eload_cfg.query('validation', 'normalisation_check', 'files', vcf_file, 'nb_realigned', ret_default=0) > 0:
+                        normalised_vcf_file.append(
+                            self.eload_cfg.query('validation', 'normalisation_check', 'files', vcf_file, 'normalised_vcf')
+                        )
+                self.eload_cfg.set('validation', 'valid', 'analyses', analysis_alias, 'vcf_files',
+                                   value=normalised_vcf_file)
```

Comment:
Won't this overwrite the valid vcf files with an empty list if none of them needed to be normalised?